pages/collections/[id].vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="flex flex-col items-center">
+    <TransitionExpand>
+      <Image v-if="isReady" url="/placeholder.png" class="aspect-[9/4] mt-8 object-cover" />
+    </TransitionExpand>
+
+    <div class="max-w-[740px]">
+      <h2 class="text-3xl md:text-5xl md:text-center mt-10 mb-6">Kolekcie</h2>
+
+      <div class="md:text-xl md:text-center">
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+        been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+        galley of type and scrambled it to make a type specimen book. It has survived not only five
+        centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+        It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum
+        passages, and more recently with desktop publishing software like Aldus PageMaker including
+        versions of Lorem Ipsum.
+      </div>
+
+      <div class="flex justify-center gap-3 my-6">
+        <div
+          class="bg-primary text-white flex items-center py-1 px-3 gap-2 cursor-pointer rounded-3xl"
+        >
+          <div class="text-xs">
+            <span class="font-bold">Autor:</span> Meno Priezvisko, Meno Priezvisko Ml.
+          </div>
+        </div>
+
+        <div
+          class="bg-primary text-white flex items-center py-1 px-3 gap-2 cursor-pointer rounded-3xl"
+        >
+          <div class="text-xs"><span class="font-bold">Technika:</span> Sietotlač, Rinolit</div>
+        </div>
+      </div>
+    </div>
+
+    <div class="w-full my-4 flex">
+      <div class="flex-grow">
+        <span class="font-bold">{{ total }}</span> děl
+      </div>
+
+      <Sort v-model:sort-by="sortBy" v-model:sort-direction="sortDirection" />
```

Comment:
Similar to #1, seeing this directly modify someone else's state doesn't seem right

```suggestion
      <Sort :by="sortBy" :direction="sortDirection" @change="({by, direction}) => ..." />
```